composer.json

```diff
@@ -10,6 +10,7 @@
     "ext-date": "*",
     "ext-dom": "*",
     "ext-iconv": "*",
+    "ext-json": "*",
```

Comment:
PhpStorm reported the lack of this in `composer.json`.

It looks like [json_decode()](https://www.php.net/manual/en/function.json-decode.php), with `AMP_HTML_Utils::is_valid_json()` calls, is from the `json` extension.

And [Jetpack also uses this](https://github.com/Automattic/jetpack/blob/3fca2c56fc064d0c1b1e180fa223feaf47e72c8c/composer.json#L12) in `composer.json`.